### PR TITLE
Apply the patch for Visual Studio 2015 on Qt 4.8.7

### PR DESCRIPTION
**Pull request by huyu398**
According to [Slicer wiki](https://www.slicer.org/wiki/Documentation/4.8/Developers/Build_Instructions#Experimental.2Fdeprecated_build_environments), we can build Slicer 4.8 with Visual Studio 2015.
However, Qt 4.8.7 isn't able to build without applying the [patch](https://stackoverflow.com/questions/32848962/how-to-build-qt-4-8-6-with-visual-studio-2015-without-official-support) on Visual Studio 2015.

This pull request enables to build Qt 4.8.7 with applying the patch on Visual Studio 2015.
I tested powershell script for Visual Studio 2015 64-bit Release and Debug.
And I succeed to build Slicer 4.8.0 with Qt 4.8.7 applying the patch.

Thank you.

### CODE REVIEW

**Comment by thewtex** [comment]
@huyu398 Thanks for contributing this! I found that I needed to download 7Zip manually, but it worked great for me with Visual Studio 2015 / Release.

**Comment by thewtex** [comment]
PS. 7Zip did not download due to an interesting TLS error.

**Comment by ihnorton** [comment]
If you are on Windows 7 you probably need: https://support.microsoft.com/en-us/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in

File: windows_build_qt.ps1 [hunk 1]
```diff
@@ -83,21 +103,29 @@ $cmake = Join-Path $cmakeInstallDir 'bin\cmake.exe'
...
 $scriptFile = Join-Path $destDir $scriptName
-$url = ('https://raw.githubusercontent.com/jcfr/qt-easy-build/' + $qtBuildScriptVersion + '/cmake/' + $scriptName)
+$url = ('https://raw.githubusercontent.com/huyu398/qt-easy-build/' + $qtBuildScriptVersion + '/cmake/' + $scriptName)
 Always-Download-File $url $scriptFile
 
 # download cross-platform helper script(s)
 $scriptName = 'QEBQt4ExternalProjectCommand.cmake'
 $scriptFile = Join-Path $destDir $scriptName
-$url = ('https://raw.githubusercontent.com/jcfr/qt-easy-build/' + $qtBuildScriptVersion + '/cmake/' + $scriptName)
+$url = ('https://raw.githubusercontent.com/huyu398/qt-easy-build/' + $qtBuildScriptVersion + '/cmake/' + $scriptName)
```
Comment (jcfr) [inline windows_build_qt.ps1:118]: could these be changed to `jcfr` ?

Comment (jcfr) [reply windows_build_qt.ps1:118]: Discard this, I will do the change now.

**Comment by jcfr** [comment]
Thanks again @huyu398 (and apologize for the very late integration)

Closing. Integrated in 4.8.7 branch

**Comment by jcfr** [comment]
> PS. 7Zip did not download due to an interesting TLS error.

> If you are on Windows 7 you probably need: https://support.microsoft.com/en-us/help/3140245/update-to-enable-tls-1-1-and-tls-1-2-as-a-default-secure-protocols-in

@thewtex @ihnorton there is a FAQ entries that should be helpful

See https://github.com/jcfr/qt-easy-build#frequently-asked-questions

**Pull request closed without merging**